server/src/api/modules/groups/services/EditGroupService.ts

```diff
@@ -266,7 +266,7 @@ async function updateMembers(groupId: number, members: Array<{ username: string;
       leftEvents.push(
         ...excessMemberships
           .filter(m => !ignoreFromLeft.includes(m.playerId))
-          .map(m => ({ playerId: m.playerId, groupId: m.groupId, type: ActivityType.LEFT }))
+          .map(m => ({ playerId: m.playerId, groupId: m.groupId, type: ActivityType.LEFT, role: m.role }))
```

Comment:
The integration test is assuming a null here since it used to be omitted from the model. Now that role is required by the models should we expect that role is added to the edit and removed responses as well, or should i force it to be null here?